app/javascript/mastodon/features/public_timeline/components/setting_text.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
I don't see any changes to this file so why not import the existing component instead of copypasting?